go.mod

```diff
@@ -15,6 +15,7 @@ require (
 
 // Test dependencies
 require (
+	github.com/golangci/golangci-lint v1.55.2
```

Comment:
Why not using the latest one `v1.56.2`?

Reply:
This change has been sitting half finished in my local git for a while. I think that must have been the latest version at the time and I did not check for newer releases.